form.html

```diff
@@ -34,8 +35,9 @@ <h3 class="how-it-works-h3">How it works</h3>
                     <dt>Let us do the rest. We are here to serve you!</dt>
                 </dl>
 
+
                 <div class="questionnaire-get-started">
-                  <button class="btn action-start">Let's get started</button>
+                  <button class="btn action-start" >Let's get started</button>

```

Comment:
Normally I would bristle at a button as a link, but in this case, I think it makes sense. A button does something, and this one starts the questionnaire.